pathplannerlib/src/main/native/include/pathplanner/lib/PathConstraints.h

```diff
@@ -3,6 +3,8 @@
 #include <units/velocity.h>
 #include <units/acceleration.h>
 
+#include <assert.h>
+
```

Comment:
This is unused, correct?

Reply:
Ah, yes, sorry. I added that while testing to see which classes are trivially copyable, since those classes are better off passed by value than reference.